frontend/components/create-review/ratings-radio.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import {RadioGroup} from '@headlessui/react'
+
+const ratings = [1, 2, 3, 4, 5]
+
+function classNames(...classes) {
+	return classes.filter(Boolean).join(' ')
+}
```

Comment:
Move this to be a utility or use https://github.com/JedWatson/classnames since we'll prob need this in multiple places

Reply:
Can do. I will create a util folder and add it in.